common/src/main/java/com/duckduckgo/app/global/ViewModelFactory.kt

```diff
@@ -20,7 +20,9 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import com.duckduckgo.app.global.plugins.view_model.ViewModelFactoryPluginPoint
 import javax.inject.Inject
+import javax.inject.Singleton
 
+@Singleton
```

Comment:
This `Singleton` does not affect the behavior.
It should have been `singleton` at the time as factory should only be one but it was fine because the `ViewModelFactoryPluginPoint` is also `singleton`